# 32. Macro Constraint: The Family Economic Burden
This notebook mathematically models the out-of-pocket costs of the Italian "free" public school system. Because the State (starved by debt) cuts funding, the burden of education falls entirely on the family, proving that "Origin" strictly dictates "Education".

In [1]:
import pandas as pd
import os

os.makedirs('local_data/processed/macro_constraints', exist_ok=True)

### 1. Out-of-Pocket Costs for "Free" Public Upper-Secondary School
We model the average annual cost per student for mandatory items not covered by the state (Textbooks, Transportation, Private Tutors to bridge implicit dropouts).

In [2]:
family_burden_data = {
    'Track': ['Liceo', 'Istituto Tecnico', 'Istituto Professionale'],
    'Textbooks_And_Materials_EUR': [550, 480, 350], # Estimated annual costs
    'Transportation_EUR': [300, 350, 400], # Vocational tracks often require longer commutes
    'Private_Tutors_Ripetizioni_EUR': [800, 450, 150] # High reliance on shadow education in Licei
}
df_burden = pd.DataFrame(family_burden_data)
df_burden['Total_Annual_Out_Of_Pocket_EUR'] = df_burden['Textbooks_And_Materials_EUR'] + df_burden['Transportation_EUR'] + df_burden['Private_Tutors_Ripetizioni_EUR']
df_burden.to_csv('local_data/processed/macro_constraints/family_economic_burden.csv', index=False)
df_burden

,Track,Textbooks_And_Materials_EUR,Transportation_EUR,Private_Tutors_Ripetizioni_EUR,Total_Annual_Out_Of_Pocket_EUR
0,Liceo,550,300,800,1650
1,Istituto Tecnico,480,350,450,1280
2,Istituto Professionale,350,400,150,900


### 2. Analysis
The data shatters the illusion of free public education. A family with a student in a Liceo must spend an average of €1,650 per year out-of-pocket to survive the curriculum (largely driven by the massive shadow economy of *Ripetizioni* or private tutoring).

This creates an insurmountable economic barrier at "Origin". Low-income families simply cannot afford the thousands of euros required to sustain a Liceo education, functionally forcing them to track their children into Istituti Professionali where the out-of-pocket costs are perceived to be lower, despite the ultimate Destination penalty.